pysatMadrigal/instruments/methods/general.py

```diff
@@ -695,6 +697,13 @@ def list_remote_files(tag, inst_id, inst_code=None, kindats=None, user=None,
     format_str = supported_tags[inst_id][tag]
     kindat = kindats[inst_id][tag]
 
+    # Note that default of `pysat.Instrument.remote_file_list` for start and
+    # stop is None. Setting defaults needed for Madrigal.
+    if start is None:
+        start = dt.datetime(1900, 1, 1)
+    if stop is None:
+        stop = dt.datetime.utcnow()
+
```

Comment:
```suggestion
```

Reply:
Why are you suggesting this be removed? Without this, the defaults aren't applied when run.

Reply:
How are they not applied?  The defaults are for these times.  Is `pysat` feeding `start` and `stop` values of `None` somehow?

Reply:
Ok, how about keeping this and adding a TODO that links to the pysat issue you just created?